src/core/message.ts

```diff
@@ -12,6 +15,29 @@ export type MessageHeaders = Headers;
  */
 export type MessageMetadata = Metadata;
 
+/**
+ * {@link DetailsMetadata} type for a chat messages {@link ActionDetails}.
+ */
+export type ActionDetailsMetadata = DetailsMetadata;
```

Comment:
Is DetailsMetadata still used anywhere else? If not, maybe just keep one type?

Reply:
It's used in the parser, so I've left it in for now :)